app/components/domain-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { AddIcon, InfoIcon } from '@chakra-ui/icons';
+import { Button, HStack, Input, Select, Textarea, Tooltip, VStack } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+
+export default function DomainForm() {
+  return (
+    <Form className="domain-form" method="post">
+      <VStack maxW="xl" spacing="5">
+        <HStack w="100%">
+          <Input placeholder="Domain Name" name="name" />
+          <Tooltip>
+            <InfoIcon></InfoIcon>
```

Comment:
If some element doesn't have any child (so nothing inside of it) we should just leave the as selfclosable tag, so
```typescript
<InfoIcon />
```